dashboard/src/actions/charts.ts

```diff
@@ -118,12 +118,17 @@ export function deployChart(
   releaseName: string,
   namespace: string,
   values?: string,
+  resourceVersion?: string,
 ) {
   return (dispatch: Dispatch<IStoreState>): Promise<{}> => {
     const chartAttrs = chartVersion.relationships.chart.data;
-    return fetch(url.api.helmreleases.create(namespace), {
+    const method = resourceVersion ? "PUT" : "POST";
```

Comment:
This is interesting, but it might be better to use a separate action for upgrade rather than using the same one to keep the intent clear. Also ideally we should move to the new way of making requests (see the Function class in `shared/Function.ts`), but we can do that in a separate PR.